chandra_aca/dark_scale.py

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
I thought that the dark scale bit was conceptually different and that the dark_model module does a bunch of math on import that you don't need if you just want a scale.